schemas/NutritionSurvey.json

```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Nutrition Survey",
+  "description": "Data collection and quality assurance tools for Nutrition Surveys on mobile devices",
+  "type": "object",
+  "properties": {
+    "doc_type": {
```

Comment:
yes [this file](https://github.com/eHealthAfrica/data-models/blob/master/schemas/index.js) sets a standard. we are just not using constants in our schemas up to now. the `doc_type` is an easy constant to start from, one day we might also have a fixed `version` within every doc, referring to a precise schema version